tests/integration/suite/actors/reminders/serialization/json.go

```diff
@@ -35,61 +35,61 @@ import (
 )
 
 func init() {
-	suite.Register(new(serializationJSON))
+	suite.Register(new(json))
```

Comment:
Conflicts with the `json` package in the standard library. Someone trying to write a test and using `json` in there will find it very confusing

Reply:
I think it is fine to use `json` as this is a struct not package and this package is not going to be imported by anyone anyway. Using the name `json` means we have no stuttering in the test name: `.../serialization/serializationJSON` vs `.../serialization/json`.

Reply:
IMHO the experience of the developer is more important than the stuttering.

If you absolutely want to avoid stuttering, consider "jsonFormat" or "encodeJSON" or something else.

Reply:
Done